Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM ubuntu:16.04
+FROM ubuntu:16.10
```

Comment:
Consider using https://hub.docker.com/_/node/ instead and removing the `apt-get install` and all the symlinking work